{"variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/identityorchestration\/tsidojourneyerror\/networkerror"]}],"schemaVersion":{"patch":0,"major":0,"minor":3},"identifier":{"url":"doc:\/\/com.TransmitSecurity.IdentityOrchestration\/documentation\/IdentityOrchestration\/TSIdoJourneyError\/networkError","interfaceLanguage":"swift"},"metadata":{"role":"symbol","title":"TSIdoJourneyError.networkError","fragments":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"networkError"}],"modules":[{"name":"IdentityOrchestration"}],"symbolKind":"case","roleHeading":"Case","externalID":"s:21IdentityOrchestration17TSIdoJourneyErrorO07networkE0yA2CmF"},"sections":[],"kind":"symbol","abstract":[{"type":"text","text":"Could not connect to server, or server did not respond before timeout."}],"hierarchy":{"paths":[["doc:\/\/com.TransmitSecurity.IdentityOrchestration\/documentation\/IdentityOrchestration","doc:\/\/com.TransmitSecurity.IdentityOrchestration\/documentation\/IdentityOrchestration\/TSIdoJourneyError"]]},"primaryContentSections":[{"kind":"declarations","declarations":[{"platforms":["iOS"],"languages":["swift"],"tokens":[{"text":"case","kind":"keyword"},{"kind":"text","text":" "},{"kind":"identifier","text":"networkError"}]}]}],"references":{"doc://com.TransmitSecurity.IdentityOrchestration/documentation/IdentityOrchestration/TSIdoJourneyError":{"type":"topic","navigatorTitle":[{"text":"TSIdoJourneyError","kind":"identifier"}],"abstract":[{"text":"The enum for the start Journey errors.","type":"text"}],"identifier":"doc:\/\/com.TransmitSecurity.IdentityOrchestration\/documentation\/IdentityOrchestration\/TSIdoJourneyError","url":"\/documentation\/identityorchestration\/tsidojourneyerror","role":"symbol","fragments":[{"text":"enum","kind":"keyword"},{"kind":"text","text":" "},{"text":"TSIdoJourneyError","kind":"identifier"}],"kind":"symbol","title":"TSIdoJourneyError"},"doc://com.TransmitSecurity.IdentityOrchestration/documentation/IdentityOrchestration/TSIdoJourneyError/networkError":{"fragments":[{"kind":"keyword","text":"case"},{"text":" ","kind":"text"},{"text":"networkError","kind":"identifier"}],"type":"topic","abstract":[{"type":"text","text":"Could not connect to server, or server did not respond before timeout."}],"identifier":"doc:\/\/com.TransmitSecurity.IdentityOrchestration\/documentation\/IdentityOrchestration\/TSIdoJourneyError\/networkError","kind":"symbol","url":"\/documentation\/identityorchestration\/tsidojourneyerror\/networkerror","title":"TSIdoJourneyError.networkError","role":"symbol"},"doc://com.TransmitSecurity.IdentityOrchestration/documentation/IdentityOrchestration":{"url":"\/documentation\/identityorchestration","title":"IdentityOrchestration","role":"collection","kind":"symbol","type":"topic","abstract":[],"identifier":"doc:\/\/com.TransmitSecurity.IdentityOrchestration\/documentation\/IdentityOrchestration"}}}